pkg/limayaml/defaults.go

```diff
@@ -814,12 +815,14 @@ func executeGuestTemplate(format, instDir string, param map[string]string) (byte
 	tmpl, err := template.New("").Parse(format)
 	if err == nil {
 		user, _ := osutil.LimaUser(false)
+		name := filepath.Base(instDir)
 		data := map[string]interface{}{
-			"Home":  fmt.Sprintf("/home/%s.linux", user.Username),
-			"Name":  filepath.Base(instDir),
-			"UID":   user.Uid,
-			"User":  user.Username,
-			"Param": param,
+			"Home":     fmt.Sprintf("/home/%s.linux", user.Username),
+			"Name":     name,
+			"Hostname": identifierutil.HostnameFromInstName(name), // TODO: support customization
```

Comment:
This new variable should be documented in `default.yaml`.

Reply:
done